lib/readline.js

```diff
@@ -1283,7 +1283,7 @@ Interface.prototype._ttyWrite = function(s, key) {
 /**
  * Creates an `AsyncIterator` object that iterates through
  * each line in the input stream as a string.
- * @returns {Symbol.AsyncIterator}
+ * @returns {symbol.AsyncIterator}
```

Comment:
```suggestion
 * @typedef {{
 *   [Symbol.asyncIterator]: () => InterfaceAsyncIterator,
 *   next: () => Promise<string>
 * }} InterfaceAsyncIterator
 * @returns {InterfaceAsyncIterator}
```

https://github.com/nodejs/node/blob/7123a00b03a908621255dc35f8d75c112901a42c/lib/internal/readline/interface.js#L1324